content/api_chef_server.md

```diff
@@ -4922,8 +4909,8 @@ The response is similar to:
 
 ``` javascript
 {
-  "_default": "https://api.opscode.com/organizations/org_name/environments/_default",
-  "webserver": "https://api.opscode.com/organizations/org_name/environments/webserver"
+  "_default": "https://api.chef.io/organizations/org_name/environments/_default",
```

Comment:
@PrajaktaPurohit Is there a reason that we use `api.opscode.com` in the API examples?